dataCollection/settings.py.template

```diff
@@ -3,7 +3,7 @@
 import os
 PROJECT_PATH = os.path.realpath(os.path.dirname(__file__))
 
-DEBUG = True
+DEBUG = False
```

Comment:
I agree for prod.  We should probably call this out in the README as something to change if running in development